documentation/modules/security/con-certificate-authorities.adoc

```diff
@@ -6,20 +6,28 @@
 = Certificate Authorities
 
 To support encryption, each {ProductName} component needs its own private keys and public key certificates.
-All component certificates are signed by a Certificate Authority (CA) called the _cluster CA_.
+All component certificates are signed by an internal Certificate Authority (CA) called the _cluster CA_.
 
-Similarly, each Kafka client application connecting using TLS client authentication needs private keys and certificates.
-The _clients CA_ is used to sign the certificates for the Kafka clients.
+Similarly, each Kafka client application connecting to {ProductName} using TLS client authentication needs to provide private keys and certificates.
+A second internal CA, named the _clients CA_, is used to sign certificates for the Kafka clients. 
 
 == CA certificates
 
-Each CA has a self-signed public key certificate.
+Both the cluster CA and clients CA have a self-signed public key certificate.
 
-Kafka brokers are configured to trust certificates signed by either the clients CA or the cluster CA. Components to which clients do not need to connect, such as ZooKeeper, only trust certificates signed by the cluster CA. Client applications that perform mutual TLS authentication have to trust the certificates signed by the cluster CA.
+Kafka brokers are configured to trust CA certificates signed by either the cluster CA or clients CA.
+Components that clients do not need to connect to, such as ZooKeeper, only trust CA certificates signed by the cluster CA.
```

Comment:
why we added "CA" ... so starting to use "CA certificates" instead of "certificates". A client should trust a broker certificates (which is not a CA certificate) signed by the cluster CA.

Reply:
I've ended up confused myself with the terminology :(

I'll remove the "CA".